x/protorev/keeper/rebalance.go

```diff
@@ -30,13 +32,10 @@ func (k Keeper) IterateRoutes(ctx sdk.Context, routes []RouteMetaData, remaining
 
 		// If the profit is greater than zero, then we convert the profits to uosmo and compare profits in terms of uosmo
 		if profit.GT(sdk.ZeroInt()) {
-			if inputCoin.Denom != types.OsmosisDenomination {
-				uosmoProfit, err := k.ConvertProfits(ctx, inputCoin, profit)
-				if err != nil {
-					k.Logger(ctx).Error("Error converting profits: ", err)
-					continue
-				}
-				profit = uosmoProfit
+			profit, err := k.ConvertProfits(ctx, inputCoin, profit)
+			if err != nil {
+				k.Logger(ctx).Error("Error converting profits: ", err)
```

Comment:
This error won't propagate properly because it accepts kv pairs, let's follow this format: https://github.com/osmosis-labs/osmosis/blob/a706848d8005f71a808adbf63d1708bc7a5b3a2c/x/protorev/keeper/posthandler.go#L61

Realized I missed all places that use the format `k.Logger(ctx).Error`, I replaced all the `ctx.Logger().Error` instances, can we add this fix in this PR too for all `k.Logger(ctx).Error`

Reply:
gotchu fam